content/cloud-docs/run/ui.mdx

```diff
@@ -63,6 +63,14 @@ If you would rather automatically apply plans that don't have errors, you can [e
 
 To help you review proposed changes, Terraform Cloud can automatically run [speculative plans][speculative plan] for pull requests or merge requests.
 
+## Upgrading Terraform using Speculative Plans
+
+Using the UI, you can start a new speculative plan using the current configuration version but any terraform version available to the organization. After clicking **Start New Run**, use the select list to choose **Plan only** as the run type. This run mode reveals a new option to choose the specific terraform version.
```

Comment:
```suggestion
Using the UI, you can start a new speculative plan with the workspace's current configuration version and any Terraform version available to the organization. After clicking **Start New Run**, use the select list to choose **Plan only** as the run type. This run mode reveals a new option to choose the specific terraform version.
```

Reply:
Can we also make the steps into a numbered list? Each item should start with a verb, and should go over one step. Be sure to tell folks what they should see when the step is completed (if appropriate). This is a best practice with ordered instructions and I think it will make this easier to read! If you take a first pass, I can edit for you :)